spec/game_spec.rb

```diff
@@ -0,0 +1,30 @@
+# spec/game_spec.rb
+
+require 'rspec'
+require_relative '../classes/game'
+
+describe Game do
```

Comment:
- [ ] An error pops while trying to verify this test, I figured it is because of the path, just as others, it should be 1 dot but you added 2, also, the rspec is not needed, I dont know why you added it but when I did it as the image below, it passed.
![image](https://github.com/George7h/Catalog-of-my-things/assets/86200356/ed17c242-6b44-4b33-b453-00be89796000)

Reply:
I have resolved the path input.